server/api/geo/index.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiEndPoint } from '@common/api/endpoint'
+import { Express } from 'express'
+
+import { getForestLayer } from './getForestLayer'
+
+export const GeoApi = {
+  init: (express: Express): void => {
+    express.get(ApiEndPoint.Geo.Layers.getForest(), getForestLayer)
```

Comment:
this ApiEndPoint.Geo.Layers.getForest() should not be invoked here.
line 8 should become:

```typescript
express.get(ApiEndPoint.Geo.Layers.getForest, getForestLayer)
```

Reply:
Sorry , I got confused 😄😀